### PR TITLE
Fix/Add options to netmap wrapper constructor

### LINKED ISSUE #793: `Netmap` Morph wrapper does not have `TryNotary` option
Add `TryNotary` option to the 4-th morph client layer (wrapper) and use it instead of 3-the client's option in IR constructor.

### PR DESCRIPTION
**Pull request by carpawell**
Closes #793.

Also fixes `errcheck` linter warning and adds `AsAlphabet` option.

### CODE REVIEW

File: pkg/morph/client/netmap/wrapper/wrapper.go [hunk 1]
```diff
@@ -22,9 +22,25 @@ type Wrapper struct {
 	client *netmap.Client
 }
 
+// Option allows to set an optional
+// parameter of ClientWrapper.
```
Comment (cthulhu-rider) [inline pkg/morph/client/netmap/wrapper/wrapper.go:26]:
```suggestion
// parameter of Wrapper.
```

Comment (cthulhu-rider) [inline pkg/morph/client/netmap/wrapper/wrapper.go:26]:
```suggestion
// parameter of Wrapper.
```

**Review by cthulhu-rider — APPROVED** [review]

File: pkg/morph/client/netmap/wrapper/wrapper.go (hunk 1 above)
Comment (carpawell) [reply pkg/morph/client/netmap/wrapper/wrapper.go:26]: do you mean to change it in every wrapper?

i copy-pasted that from other contract wrappers

Comment (cthulhu-rider) [reply pkg/morph/client/netmap/wrapper/wrapper.go:26]: Type is named `Wrapper`, so the doc should follow it.

**Review by alexvanin — APPROVED** [review]

File: pkg/morph/client/netmap/wrapper/wrapper.go (hunk 1 above)
Comment (carpawell) [reply pkg/morph/client/netmap/wrapper/wrapper.go]: ok, rewrited commentary and add commit that fixes the same for other wrappers

**Pull request merged**